tests/test_mcp.py

```diff
@@ -0,0 +1,240 @@
+import pytest
+import re
+import time
+import sys
+import io
+import asyncio
+import threading
+import json
+from agentic.common import Agent, AgentRunner
+from agentic.tools.mcp_tool import MCPTool
+from typing import List, Any, Callable
```

Comment:
```suggestion
import pytest
import json
from agentic.common import Agent, AgentRunner
from agentic.tools.mcp_tool import MCPTool
```

Lots of unused imports